nodejs/aws-infra/network.ts

```diff
@@ -101,8 +101,10 @@ export class Network extends pulumi.ComponentResource implements ClusterNetworkA
         if (!defaultNetwork) {
             const vpc = aws.ec2.getVpc({default: true});
             const vpcId = vpc.then(v => v.id);
-            const subnetIds = aws.ec2.getSubnetIds({ vpcId: vpcId }).then(subnets => subnets.ids);
-            const defaultSecurityGroup = aws.ec2.getSecurityGroup({ name: "default", vpcId: vpcId }).then(sg => sg.id);
+            const subnetIds = vpcId.then(id => aws.ec2.getSubnetIds({ vpcId: id })).then(subnets => subnets.ids);
```

Comment:
This responds to https://github.com/pulumi/pulumi-terraform/commit/720f2325981df3411ad97437dc6c909f119fe919